test/test_modules.py

```diff
@@ -156,6 +156,95 @@ def test_pickle(self, device, dtype, module_info):
                     output_from_copy = m_copy(*args, **kwargs)
                     self.assertEqual(output, output_from_copy)
 
+    @modules(module_db, allow_devices={"cuda"})
+    def test_cpu_gpu_results_are_equal(self, device, dtype, module_info):
+        # Test cpu and gpu results are the same
+        module_cls = module_info.module_cls
+        module_inputs_cpu = module_info.module_inputs_func(module_info, device="cpu", dtype=dtype,
+                                                           requires_grad=True)
```

Comment:
I think it's a little more "standard" to create the tensors on a cpu and move to a gpu, but I am okay either way.